lib/jira/api.rb

```diff
@@ -25,23 +25,24 @@ def delete(path, options={})
 
     def process(response, options)
       raise UnauthorizedException if response.status == 401
-      json = response.body || {}
+      body = response.body || {}
+      json = (body if body.class == Hash) || {}
```

Comment:
hmm - good to know. i think we can probably find a cleaner solution than this, but it would require better handling of the response from the API class. fine for now, but would be good to clean this up at some point